docs/extend/custom-components-remotes.md

```diff
@@ -12,10 +12,10 @@ description: "Implement custom components and register them on a server configur
 ---
 
 {{% alert title="Caution" color="caution" %}}
-{{< glossary_tooltip term_id="module" text="Modular resources" >}} are the preferred method of creating custom resource implementations for SDKs with module support unless you are hosting `viam-server` on a non-Linux system or have another issue with compilation.
+[Modular resources](/extend/modular-resources/key-concepts/) are the preferred method of creating custom resource implementations for SDKs with module support unless you are hosting `viam-server` on a non-Linux system or have another issue with compilation.
```

Comment:
Why don't we instead change the glossary entry to link to that 😊

Reply:
Right on, I'll do that tomorrow. 👍